index.ts

```diff
@@ -161,18 +161,32 @@ export default class AuditLogPlugin extends AdminForthPlugin {
       }
 
       resource.options = resource.options || {} as any;
-      resource.options.pageInjections = resource.options.pageInjections || {} as any;
-      resource.options.pageInjections.show = resource.options.pageInjections.show || {} as any;
-      if (!resource.options.pageInjections.show.bottom) {
-        resource.options.pageInjections.show.bottom = [] as any[];
-      } else if (!Array.isArray(resource.options.pageInjections.show.bottom)) {
-        resource.options.pageInjections.show.bottom = [resource.options.pageInjections.show.bottom] as any[];
+      if (!resource.options.actions) {
+        resource.options.actions = [];
       }
-      {
-        const bottom = resource.options.pageInjections.show.bottom as any[];
-        const compDecl = { file: this.componentPath('RelatedLogsLink.vue'), meta: { ...this.options, pluginInstanceId: this.pluginInstanceId, auditLogResourceId: this.auditLogResource, ADMIN_BASE_URL: (this.adminforth as any)?.config?.baseUrlSlashed || '' } } as any;
-        const already = bottom.some((d: any) => d?.file === compDecl.file);
-        if (!already) bottom.push(compDecl);
+      
+      const historyActionId = 'audit_log_history_btn';
+      if (!resource.options.actions.find((a) => a.id === historyActionId)) {
+        resource.options.actions.push({
```

Comment:
@tesaide interesting but I cant find `if (resource.resourceId !== this.auditLogResource) {` check here, means link will be on logs resurce also?)) or am I missing it? Can you point me please?